dcpy/models/geospatial/projection.py

```diff
@@ -450,16 +448,15 @@ class TemporalCrs(BaseModel, extra="forbid"):
     ids: Any | None = None
 
 
-class Datum(RootModel):
-    root: (
-        GeodeticReferenceFrame
-        | VerticalReferenceFrame
-        | DynamicGeodeticReferenceFrame
-        | DynamicVerticalReferenceFrame
-        | TemporalDatum
-        | ParametricDatum
-        | EngineeringDatum
-    )
+Datum: TypeAlias = (
```

Comment:
RootModel seems to be Pydantic's preferred way of dealing with this sort of thing, but it was a bit annoying to work with